### DISCUSSION TITLE
What is the difference between RiMusic UI and ViMusic UI?

### DISCUSSION THREAD

**Discussion by privacyfreak84** [fast4x/RiMusic#1581] [Q&A]
This might be a stupid question, but I think it would be nice to have a list of changes between the 2 UIs since there have been so many changes. Are there any features that are specific to the RiMusic UI?

The last part of the question is effectively the reason I created this discussion. I now prefer the ViMusic UI, and in using it, don't want to miss out on any features

**Comment by fast4x** [comment]
ViMusic
- not have new top bar navigation 
- title pages move to the right edge
- fixed icon search at the bottom right 
- in player titles are centered
- in player not have same actions buttons to the bottom

**Reply by privacyfreak84** [reply]
So only the extra actions in the player are missing?
